specification/metrics/effectivecost.md

```diff
@@ -1,13 +1,13 @@
 # Effective Cost
 
-Effective cost represents the sum of the amortized upfront fees, amortized recurring fees, and the usage cost, including discounts,  of the line item. This column is often used to show spending trends that include on demand rates, effective commitment-based discount rates, recurring commitments, as well as amortization of upfront fees. The fee and upfront amortization portion of this column's value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Effective Cost can be found in the [Billing Currency](#billingcurrency) column. The aggregated Effective Cost for a billing period may not match the charge received on the invoice for the same billing period.
+The Effective Cost represents a charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends. The amortization of upfront charges portion of this metric's value should be proportional to the [Quantity In Pricing Unit](#quantityinpricingunit) and the time granularity of the data.
 
 Practitioners are faced with two main challenges with respect to this metric:
 
 1. Practitioners need to amortize upfront fees over the duration of the commitment and distribute those fees to the appropriate reporting groups (e.g. tags, resources).
 2. Many commitment-based discount constructs include a recurring expense for the commitment for every billing period and must distribute this cost to the resources using the commitment. This forces reconciliation between the initial commitment line item per period and the actual usage line items.
 
-The EffectiveCost column MUST be present in the billing data. This column MUST be a numeric value of type Decimal and MUST NOT contain null values.
+The EffectiveCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal and MUST NOT contain null values. The aggregated EffectiveCost for a billing period MAY NOT  match the charge received on the invoice for the same billing period.
```

Comment:
Removing an extra space.

nit: Does the added sentence provide any value? If it's a note for practitioners, then it should probably be above. If it's a requirement for providers (which is the purpose of this paragraph), then it doesn't commit them to anything. It just acknowledges that the numbers may not match. Am I missing the key takeaway? 
```suggestion
The EffectiveCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal and MUST NOT contain null values. The aggregated EffectiveCost for a billing period MAY NOT match the charge received on the invoice for the same billing period.
```